### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vitvadym**
[DEMO LINK](https://vitvadym.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/blocks/catalog.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.catalog {
+  display: grid;
+  margin: 0 auto;
+  max-width: 944px;
+
+  gap: $card-gap;
+  justify-items: center;
+
+  @media (min-width:488px) {
+    grid-template-columns: repeat(2,1fr);
```
Comment (VVBanas) [inline src/blocks/catalog.scss:10]: Proposal: rows 10, 14, 18. add space before 1 fr.

File: src/utils/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+$card-width: 200px;
+$card-height: 300px;
+$card-gap:48px;
```
Comment (VVBanas) [inline src/utils/_variables.scss:3]: Proposal: add space after $card-gap: .

File: src/blocks/catalog.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.catalog {
+  display: grid;
+  margin: 0 auto;
+  max-width: 944px;
+
+  gap: $card-gap;
+  justify-items: center;
+
+  @media (min-width:488px) {
```
Comment (VVBanas) [inline src/blocks/catalog.scss:9]: Proposal: add space after min-width: .

**Review by VVBanas — CHANGES_REQUESTED** [review]
I like your solution. : )

**Review by silchyn — APPROVED** [review]
Pay attention to the comments above

**Pull request closed without merging**